auth/user_mgt.go

```diff
@@ -416,6 +427,13 @@ func validatePhone(val interface{}) error {
 	return nil
 }
 
+func validateValidSince(val interface{}) error {
```

Comment:
Given that there's no way for a user to set this explicitly, I wonder if this check is even needed. Consider removing.